### PR TITLE
navbar: added navbar to AppLayout for app-wide access.

### PR DESCRIPTION
**Pull request by sethdivyansh**
fixes: #121

## Description
added navbar to AppLayout for app-wide access.

## Checklist
- [x] I have tested these changes locally.
- [ ] I have reviewed the code and ensured it follows the project's coding guidelines.
- [ ] I have updated the documentation, if necessary.
- [ ] I have assigned reviewers to this pull request.

## Screenshots (if applicable)
[If your changes include any visual updates, provide screenshots here.]

### CODE REVIEW

File: frontend/src/pages/Home.tsx [hunk 1]
```diff
@@ -22,7 +22,7 @@ const Home: React.FC = () => {
 
     return (
         <div className="min-h-screen bg-uno-bg bg-cover bg-center flex flex-col relative">
-            <Navbar />
+            {/* <Navbar /> */}
```
Comment (kuv2707) [inline frontend/src/pages/Home.tsx:25]: Why not just delete the line?

Comment (kuv2707) [reply frontend/src/pages/Home.tsx:25]: Also, remove the occurrence of navbar from all the other places

Comment (sethdivyansh) [reply frontend/src/pages/Home.tsx]: done

**Comment by kuv2707** [comment]
Looks good in manual testing

**Comment by kuv2707** [comment]
Merged, thanks @sethdivyansh

**Pull request merged**